### PR TITLE
[16.0][MIG] sale_order_report_product_image: Migration to 16.0

### PR DESCRIPTION
**Pull request by JesusValdez96**

### CODE REVIEW

File: sale_order_report_product_image/i18n/sale_order_report_product_image.pot [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* sale_order_report_product_image
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 15.0\n"
```
Comment (hugho-ad) [inline sale_order_report_product_image/i18n/sale_order_report_product_image.pot]: could you update pot files

**Review by hugho-ad — APPROVED** [review]
in genneral LGTM :+1:

File: sale_order_report_product_image/i18n/sale_order_report_product_image.pot (hunk 1 above)
Comment (JesusValdez96) [reply sale_order_report_product_image/i18n/sale_order_report_product_image.pot]: @hugho-ad Updated, please review again.

**Comment by hugho-ad** [comment]
@moylop260 @luisg123v

could you review pls?

**Comment by moylop260** [comment]
Could you add a video testing the feature in runboat, please?

**Review by luisg123v — APPROVED** [review]
LGTM :+1: 

/ocabot migration sale_order_report_product_image

**Comment by luisg123v** [comment]
@moylop260 could you run the migration command by me, please?

**Comment by JesusValdez96** [comment]
@moylop260 Could you review again?

**Pull request merged**